File: jobsheet1.cpp
#include <iostream>
using namespace std;

int main(){
    

    cout << "tugas 1 \n";
    int i;

    cout << "tugas 1.if-else \n";

    cout << "Makanan apa yang anda suka? \n 1. Ayam Goreng \n 2. Bebek Goreng\n 3. Lele Goreng\n Pilih = ";
    cin >> i;

    if (i == 1){
        cout << "Ayam Goreng \n";
    } else if (i == 2)
    {
        cout << "bebek goreng \n";
    } else if (i == 3)
    {
        cout << "lele goreng \n";
    } else {
        cout << "tidak suka semua \n";
    }

    cout << "tugas 1.switch-case \n";

    cout << "Makanan apa yang anda suka? \n 1. Ayam Goreng \n 2. Bebek Goreng\n 3. Lele Goreng\n Pilih = ";
    cin >> i;

    switch (i)
    {
    case 1:
        cout << "Ayam goreng \n";
        break;
    case 2:
        cout << "Bebek goreng \n";
        break;
    case 3:
        cout << "Lele goreng \n";
        break;
    default:
        cout << "Tidak Suka Semua\n";
        break;
    }

    cout << "tugas 2\n";
    int umur, status;

    cout << "tugas 2.if-else\n";
    cout << "Berapakah umur anda? ";
    cin >> umur;

    if (umur >= 18)
    {
        cout << "Apakah anda sudah terdaftar?\n ketik 1 apabila sudah dan 2 apabila belum\n";
        cin >> status;
        if (status == 1)
        {
            cout << "Anda berhak memilih\n";
        }else{
        cout << "Anda belum terdaftar\n";
        }
        
    }else{
    cout << "Anda belum berhak memilih \n";
    }

    cout << "tugas 2.switch-case\n";
    cout << "apakah umur anda sudah 18 tahun keatas?\n 1. sudah\n 2. belum\n";
    cin >> umur;

    switch (umur)
    {
    case 1:
        cout << "Apakah anda sudah terdaftar?\n ketik 1 apabila sudah dan 2 apabila belum\n";
        cin >> status;
        switch (status)
        {
        case 1:
            cout << "Anda berhak memilih\n";
            break;
        default:
            cout << "Anda belum terdaftar\n";
            break;
        }
        break;
    
    default:
        cout << "Anda belum berhak memilih\n";
        break;
    }

    


}

File: jobsheet1.py
from typing import Match


print ("tugas 1\n")
print ("tugas 1.elif\n")

print("Makanan apa yang anda suka?\n 1. Ayam Goreng\n 2. Bebek Goreng\n 3. Lele Goreng\n")

menu = int(input("Pilihan anda : "))

if (menu ==1):
    print("Ayam Goreng")
elif (menu ==2):
    print("Bebek Goreng")
elif (menu ==3):
    print("Lele Goreng")
else:
    print("Tidak suka semua")

print("=" * 50)

print ("tugas 1.dictionary\n")
print("Makanan apa yang anda suka?\n 1. Ayam Goreng\n 2. Bebek Goreng\n 3. Lele Goreng\n")

def menu(i):
    switcher={
        1: "Ayam Goreng",
        2: "Bebek Goreng",
        3: "Lele Goreng"
    }
    return switcher.get(i, "Tidak Suka Semua")

menu_favorit = menu(int(input("Pilihan anda : ")))
print(menu_favorit)

print("=" * 50)

print("tugas 2\n")
print("tugas 2.elif\n")

print("Berapakah Umur Anda Saat Ini?\n")
umur = int(input("Umur Saya : "))
#use boolean is not effective
status = bool()

if umur >= 18:
    print("Apakah Anda Sudah Terdaftar?\n1. Sudah\n2. Belum\n")
    status = int(input("Status Saya : "))
    if status == True:
        print("Anda Berhak Memilih\n")
    else:
        print("Anda Belum Terdaftar\n")
else:
    print("Anda Belum Berhak Memilih\n")

print("=" * 50)

# print("tugas 2.dictionary\n")

# print("Apakah Umur Anda Sudah 18 Tahun Keatas?\n1. Sudah\n2. Belum\n")
# umur = int(input("Umur Saya : "))
# def memilih(i):
#     switcher={
#         True: {print("Apakah Anda Sudah Terdaftar?\n1. Sudah\n2. Belum\n")
#     status = int(input("Status Saya : "))}
#     }
